auslib/blobs/schemas/apprelease-v6.yml

```diff
@@ -0,0 +1,210 @@
+---
```

Comment:
You'll need to add support for "product" here, as I just merged https://github.com/mozilla/balrog/pull/129.